src/NServiceBus.Core/Serialization/ConfigureSerialization.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace NServiceBus.Serialization
+{
+    using System;
+    using NServiceBus.Features;
+    using NServiceBus.MessageInterfaces.MessageMapper.Reflection;
+    using NServiceBus.Serializers;
+
+    /// <summary>
+    /// Base class for configuring <see cref="SerializationDefinition"/> features.
+    /// </summary>
+    public abstract class ConfigureSerialization : Feature
+    {
+        /// <inheritdoc />
+        protected ConfigureSerialization()
+        {
+            EnableByDefault();
+            Prerequisite(context => this.IsDefaultSerializer(context) || this.IsAdditionalDeserializer(context), 
```

Comment:
@danielmarbach yes, unfortunately, it's an extension method on `ConfigureSerialization`. Not sure it should be, but the previous one was :D